src/utils/elderly/profile.ts

```diff
@@ -2,6 +2,6 @@ import { client } from '../../config/axiosConfig';
 import { ElderlyHomeProfile } from '../../dto/modules/modules.dto';
 
 export async function getElderlyProfile(): Promise<ElderlyHomeProfile> {
-  const { data } = await client.get('home/elderlyHome');
+  const { data } = await client.get('/home/elderlyHome');
   return data as ElderlyHomeProfile;
 }
```

Comment:
Consider doing error handling? If the fetch results in an error (like jwt expired), then you can show a toast message in case of error. Sorry, I haven't created the design yet. So, maybe you can add it in later PR. No problem for this one. I and Note will add the design for all error cases in the future sprint.

Reply:
There are other places without error messages prepared from other pages as well. We will update the design for those cases na. For this PR it's ok, go ahead first.